docs/kiali/02-01-kiali.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Architecture
```

Comment:
For the architecture part - it's clear that we have two parts here:
- how the user uses Kiali
- how it works under the hood
I was wondering if maybe we can have two diagrams for that to show the user flow, and then to show how it basically works underneath (in a little bit more detail than described right now).
What do you think? I already started working on the diagram rework, so maybe I can suggest how it can be divided - I assume the backend needs to be set up for the user to work with Kiali, so we can start with that, meaning how it is deployed when, and how it works and then describe how I, as the user, can play around.
What do you think? @clebs , @a-thaler

Reply:
Hi @bszwarc,
yes sure! If you have ideas on how to better illustrate this, please go ahead. And yes, Kiali is installed with Kyma as a component and integrated into dex so that users have access to it.

Reply:
I think we should keep one diagram, it outlines all involved components nicely. It should not explain how the user uses kiali but how the components interact with each other.

Reply:
I would vote for 2 diagrams still - one about the process, second about the backend. However, I tried to put it all in one diagram for now. For me it is still more of a flow and some elements underneath than just enumeration of components :)